API.md

```diff
@@ -935,6 +935,17 @@ exports.register.attributes = {
 };
 ```
 
+### `server.requestIdGenerator(method)`
+
+Registers a custom generator for request IDs where:
+- `method` - the function that is invoked using the signature `function(request)` and returns an ID, or `null` if the default should be used
+
+```js
+server.requestIdGenerator((request) => {
```

Comment:
Hi @hueniverse - my additional push crossed paths with your message suggesting this. I attempted to address this in this comment. Would appreciate any thoughts on how I could make the route config work, as I'd also prefer that.